frontend/app/config/configuration-service.js

```diff
@@ -90,5 +90,11 @@ module.exports = function() {
     isModuleEnabled: function(module) {
       return this.settings.enabled_modules.indexOf(module) >= 0;
     },
+    startOfWeekPresent: function() {
+      return this.displaySettingPresent('start_of_week') && this.setting.display.start_of_week != '';
```

Comment:
this was introduced by @hschink intiall, nothing @NobodysNightmare , @hirtie-maxim or myself touched.